<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<pysystest state="runnable" type="auto">
	<description>
		<title>Spread 001: Config validation errors.</title>
		<purpose>
		<![CDATA[Config validation errors.]]>
		</purpose>
	</description>
	<classification>
		<groups>
			<group>Spread</group>
			<group>Streaming_Calculations</group>
			<group>Industry_Analytics</group>
			<group>EPL</group>
		</groups>
	</classification>
	<data>
		<class module="run" name="PySysTest"></class>
	</data>
	<traceability>
		<requirements>
			<requirement></requirement>
		</requirements>
	</traceability>
</pysystest>